tchannel/event.py

```diff
@@ -62,6 +63,12 @@ def before_send_request(self, request):
 
     """
 
+    def before_send_request_headers(self, headers):
```

Comment:
`before_serialize_request_headers`?  seems confusing when named "send" but commented everywhere that it's about serialization.